typescript/ecs/fargate-service-with-logging/index.ts

```diff
@@ -0,0 +1,43 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import cdk = require('@aws-cdk/cdk');
+
+class WillkommenFargate extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // For better iteration speed, it might make sense to put this VPC into
+    // a separate stack and import it here. We then have two stacks to
+    // deploy, but VPC creation is slow so we'll only have to do that once
+    // and can iterate quickly on consuming stacks. Not doing that for now.
+    const vpc = new ec2.VpcNetwork(this, 'MyVpc', { maxAZs: 2 });
+    const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
+
+    // create a task definition with CloudWatch Logs
+    const logging = new ecs.AwsLogDriver(this, "AppLogging", {
+        streamPrefix: "myapp",
+    })
+    const taskDef = new ecs.TaskDefinition(this, "MyTaskDefinition", {
+        compatibility: ecs.Compatibility.Fargate,
+        memoryMiB: '512',
+        cpu: '256',
+    })
+    taskDef.addContainer("AppContainer", {
+        image: ecs.ContainerImage.fromDockerHub("amazon/amazon-ecs-sample"),
+        logging,
+    })
+
+    // Instantiate ECS Service with just cluster and image
+    new ecs.FargateService(this, "FargateService", {
+      cluster,
+      taskDefinition: taskDef,
```

Comment:
I'm wondering if we need both Fargate and ECS examples here since they're both the same? If I were to choose one, maybe just the Fargate example?

/c @clareliguori @rix0rrr